### ISSUE TITLE
The Display Name and Service URL fields are required, but the visual indication for this information is missing, and NVDA/JAWS is not announcing these fields as required.: A11y_AzureToolsStorageExplorer_Enter Connection Info_ScreenReader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8417]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #WCAG3.3.2; #FTP; #Label and Instructions; #NVDA; #JAWS; #STP;

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
Screen reader
NVDA Version: 2024.4.1
JAWS Version: 2024.2312.53

## Repro Steps

1. Turn on NVDA screen reader
2. Launch Azure Storage Explorer application.
3. TAB till "Open Connect Dialog" present on left side of screen and hit enter.
4. Select "Storage account or service", then TAB till Next button and press ENTER key.
5. In "Select Connection Method", select radio button "Shared access signature URL (SAS)".
6. TAB till Next button and press ENTER key.
7. TAB till Display name and Service URL edit filed.
8. Observe that the "Display Name" and "Service URL" fields are required, but the visual indication for this information is missing, and NVDA is not announcing these fields as required.

## Actual Experience

The "Display Name" and "Service URL" fields are required, but the visual indication for this information is missing, and NVDA is not announcing these fields as required.
Note: Issue is repro with JAWS also

## Expected Experience
There should be visual information indicating that both edit fields are required, and NVDA should also announce as required when we tab to them.

## User Impact:
User will not know "Display Name" and "Service URL" fields are required because visual indication for this information is missing, and NVDA is not announcing these fields as required.

## Attachment
![Image](https://github.com/user-attachments/assets/9e45d812-61d8-4c90-9655-1de18e5f08cb)

JAWS
![Image](https://github.com/user-attachments/assets/4dd7aa63-e85b-41f1-8543-dc01a58e7ba0)

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by kapilvaishna** [comment]
Issue is repro in 1.37.0 version, please find repro attachment below 

![Image](https://github.com/user-attachments/assets/529b2dff-64a4-49f9-a7ef-6525209374ec)

![Image](https://github.com/user-attachments/assets/8e74ee0a-2379-4a53-a96d-6ee2057b32b5)

**Comment by craxal** [comment]
@kapilvaishna The display name is not strictly required, because it is auto generated if the URL is filled out without modifying the display name manually. We can perhaps rework the UI to reflect this.

**Comment by kapilvaishna** [comment]
@craxal if display name is auto generated than also if marked as required and NVDA announce it required than also there is no issue because the requirement is met regardless of whether the edit field is filled or not.

**Comment by craxal** [comment]
@kapilvaishna Fix is ready for validation.

**Comment by kupatkar99** [comment]
@craxal Verified the issue in below Environment:
Azure Storage Explorer version:1.38.0-dev(104)
Issue is not fixed. Please find below added attachment for reference.

![Image](https://github.com/user-attachments/assets/e44e50e3-1e84-48c2-a319-288a46c4e88e)
